src/models/reposyncstate.ts

```diff
@@ -34,17 +34,20 @@ export interface RepoSyncStateProperties {
 	pullStatus?: TaskStatus;
 	buildStatus?: TaskStatus;
 	deploymentStatus?: TaskStatus;
+	dependabotAlertStatus?: TaskStatus;
```

Comment:
Have you run the db migration scripts yet?

Reply:
Yes